.eslintignore

```diff
@@ -0,0 +1,4 @@
+#eslint will ignore these files
```

Comment:
`/.eslintignore` or `/exercises/grains/.eslintignore` is probably redundant. I would favor the one on `/exercises/grains/`, because it's more specific.

Have you tried with only one of these files?